examples/drupal/README.md

```diff
@@ -62,9 +62,9 @@ lando composer --version | grep Composer | grep 2.
 cd drupal9
 lando ssh -c "/usr/local/bin/composer --version" | grep Composer | grep "2.0.11"
 
-# Should use php version 7.4 by default
+# Should use php version 8.2 by default
 cd drupal9
-lando php -v | grep "PHP 8.1"
+lando php -v | grep "PHP 8.2"
```

Comment:
I believe this is what is failing the tests, since the .lando.yml in the Acquia repo probably specifies the older version.

Reply:
@reynoldsalec On the drupal* repo?

Reply:
Na, this test actually pulls down a Drupal site from Acquia, so in the repo for that project on Acquia I'm guessing it specifies PHP 8.1